irods_testing_environment/odbc_setup.py

```diff
@@ -15,55 +15,90 @@ def configure_postgres_odbc_driver(csp_container, odbc_driver):
+    odbc_driver -- path to local archive file containing the ODBC driver package
+    """
+    configure_postgres_odbc_driver(csp_container, odbc_driver)
+
+def configure_odbc_driver_ubuntu_2404_postgres_14(csp_container, odbc_driver):
+    """Configure ODBC driver for postgres 14 on ubuntu 24.04.
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
     """
     configure_postgres_odbc_driver(csp_container, odbc_driver)
 
-def configure_odbc_driver_ubuntu_2004_postgres_148(csp_container, odbc_driver):
-    """Configure ODBC driver for postgres 14.8 on ubuntu 20.04.
+def configure_odbc_driver_ubuntu_2404_postgres_16(csp_container, odbc_driver):
+    """Configure ODBC driver for postgres 16 on ubuntu 24.04.
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
     """
     configure_postgres_odbc_driver(csp_container, odbc_driver)
 
-def configure_odbc_driver_ubuntu_2204_postgres_148(csp_container, odbc_driver):
-    """Configure ODBC driver for postgres 14.8 on ubuntu 22.04.
+def configure_odbc_driver_debian_11_postgres_14(csp_container, odbc_driver):
+    """Configure ODBC driver for postgres 14 on debian 11.
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
     """
     configure_postgres_odbc_driver(csp_container, odbc_driver)
 
-def configure_odbc_driver_ubuntu_2404_postgres_148(csp_container, odbc_driver):
-    """Configure ODBC driver for postgres 14.8 on ubuntu 24.04.
+def configure_odbc_driver_debian_11_postgres_16(csp_container, odbc_driver):
+    """Configure ODBC driver for postgres 16 on debian 11.
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
     """
     configure_postgres_odbc_driver(csp_container, odbc_driver)
 
-def configure_odbc_driver_debian_11_postgres_1012(csp_container, odbc_driver):
-    """Configure ODBC driver for postgres 10.12 on debian 11.
+def configure_odbc_driver_debian_12_postgres_14(csp_container, odbc_driver):
+    """Configure ODBC driver for postgres 14 on debian 12.
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
     """
     configure_postgres_odbc_driver(csp_container, odbc_driver)
 
-def configure_odbc_driver_debian_12_postgres_148(csp_container, odbc_driver):
+def configure_odbc_driver_debian_12_postgres_16(csp_container, odbc_driver):
     """Configure ODBC driver for postgres 14.8 on debian 12.
```

Comment:
postgres 14.8 doesn't match function name of postgres 16

Reply:
Fixed